br/pkg/backup/client.go

```diff
@@ -410,7 +410,16 @@ func (bc *Client) BuildBackupRangeAndSchema(
 	isFullBackup bool,
 ) ([]rtree.Range, *Schemas, []*backuppb.PlacementPolicy, error) {
 	if bc.checkpointMeta == nil {
-		return BuildBackupRangeAndSchema(storage, tableFilter, backupTS, isFullBackup, true)
+		ranges, schemas, policies, err := BuildBackupRangeAndSchema(storage, tableFilter, backupTS, isFullBackup, true)
```

Comment:
Both named as` BuildBackupRangeAndSchema` is a little confusing, can we rename one?

Reply:
done. Remed as `BuildBackupRangeAndInitSchema`